MagazineLayout/Public/Types/MagazineLayoutItemSizeMode.swift

```diff
@@ -132,7 +132,7 @@ public enum MagazineLayoutItemHeightMode: Hashable {
   /// For example, if you support multiline labels or dynamic type, your height is likely not known
   /// until the Auto Layout engine resolves the layout at runtime.
   public static var dynamic: MagazineLayoutItemHeightMode {
-    .dynamic(estimatedHeight: MagazineLayout.Default.ItemHeight)
```

Comment:
See we still use this constant elsewhere, should we change the constant value instead? A bit confused on when we want this smaller height versus not.

Reply:
I realized we were using this default item height for not just the estimated height, but also the default static height if someone doesn't implement `sizeModeForItemAt`. Didn't want to cause a regression in that case.